### PR TITLE
Implemented Inventory Service 

### PR DESCRIPTION
**Pull request by raaedk**

### CODE REVIEW

**Comment by GannonDeWalt** [comment]
we pulled from master to see what was up, but it looks like it deleted the json file we had under the data folder and that is the issue. Trying to fix now

**Comment by raaedk** [comment]
so the entire inventory service got yeeted off the face of the planet when we git pulled so dont merge this yet

**Comment by raaedk** [comment]
this is done now

**Review by rmuehler — APPROVED** [review]
Looks good to me.

**Review by ammitchky — APPROVED** [review]
looks good 2.0

File: src/app/tests/inventory.service.spec.ts [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    expect(service).toBeTruthy();
+  });
+
+  it('sold() should delete the vehicle from inventory', () => {
+    expect(service).toBeTruthy();
+  });
+
+  it('addVehicle() should add a vehicle to the inventory', () => {
+    expect(service).toBeTruthy();
+  });
```
Comment (wangscs) [inline src/app/tests/inventory.service.spec.ts:64]: tests should be commented out if not correct

File: src/app/tests/vehicle.spec.ts [hunk 2]
```diff
@@ -1,7 +1,16 @@
-import { Vehicle } from '../models/vehicle';
+import Vehicle, { convertVehicle } from '../models/vehicle';
+
```
Comment (wangscs) [inline src/app/tests/vehicle.spec.ts:2]: unnecessary new lines should be handled before pushing

**Review by wangscs — APPROVED** [review]
Looks good!

**Pull request merged**